src/translations/en.json

```diff
@@ -559,8 +559,6 @@
     "ImportDephasingRule":"Import dephasing rule",
 
     "GenerateSkeleton": "Generate CSV skeleton",
-    "UploadCSV": "Uploadd CSV",
```

Comment:
Upload, with only one 'd' at the end

Reply:
This locale was not needed so I removed it